Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.cpp

```diff
@@ -969,6 +974,55 @@ namespace AZ
             return AZStd::chrono::duration_cast<AZStd::chrono::microseconds>(timeInNano);
         }
 
+        AZStd::pair<uint64_t, uint64_t> Device::GetCalibratedTimestamp([[maybe_unused]] RHI::HardwareQueueClass queueClass)
+        {
+            if (!static_cast<const PhysicalDevice&>(GetPhysicalDevice())
```

Comment:
Do we want to cache this in a boolean within DeviceFeatures instead of calling IsOptionalDeviceExtensionSupported each time?

Reply:
`IsOptionalDeviceExtensionSupported` internally only does a `test` on a `Bitset`, the difference is negligible here in my opinion.